### DISCUSSION TITLE
How set custom claims on jwt?

### DISCUSSION THREAD

**Discussion by isomorpheric** [pocketbase/pocketbase#2525] [Q&A] [answered]
I come from a Firbase background, and for RBAC I can set custom claims on tokens (like roles: ['accounts', 'sales']) 

Is that something we can do?

**Comment by ganigeorgiev** [comment, marked as answer]
No, we keep in the payload only the fields that we know that can't change between requests.

Note that even if we allow specifying custom payload fields, you still will have to send a request to the backend server to validate the token.

If you want to get an up-to-date user data, you can call `authRefresh()`:

```js
await pb.collection("users").authRefresh();
```

**Reply by isomorpheric** [reply]
Understood - thank you for your work.
